docs/src/clusters.md

```diff
@@ -36,15 +36,15 @@ solana config set --url https://devnet.solana.com
 
 ```bash
 $ solana-validator \
+    --entrypoint entrypoint.devnet.solana.com:8001 \
```

Comment:
Reordered for logical use order: entrypoint (contact to the cluster) => [trusted] validator (fetch genesis/snapshot) => expected-... (let's assert expected things finally)